### DISCUSSION TITLE
Messages are beign processed with the wrong queue processor

### DISCUSSION THREAD

**Discussion by amine-y** [rabbitmq/rabbitmq-server#4989] [Questions]
I have a problem with Magento 2 custom queue processor sometimes the messages that are being sent to topic 1 are being processed by the topic 2 queue processor is there's any solution for that problem

Thanks,

**Comment by michaelklishin** [comment]
This is a question for a Magento forum. They will probably ask for an executable way to reproduce or at least some relevant code/configuration. We cannot suggest anything from a two sentence problem definition, I'm afraid.

**Reply by amine-y** [reply]
Thanks for your reply,
I just want to verify that RabbitMQ is not responsible for that behaviour and this is a Magento2 problem so I can follow the problem in magento forum can you give me a clear response on that?
